index.html

```diff
@@ -17,5 +17,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    
+    <header>
+        
+        <nav>
+           <ul class="list">
+                <li><img src="img/karma-logo.svg" alt="karma logo"/></li>
+                <li><a href="#"><span>Meet Karma</span></a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="section1">
+            
+            <h1>Introducing Karma</h1>
+            <h2>Bring wifi with you, everywhere you go.</h2>
+            <button class="button">Learn More</button>
+        </section>
+        <section class="section2">
+            <h1 class="icon-title">Everyone needs a little Karma</h1>
+            <div class="divs">
```

Comment:
I would avoid using a class name like this. Try to be more specific to the content within the divs. i.e 'icons' or 'info-icons'